packages/core/src/components/toast/toastProps.ts

```diff
@@ -47,7 +47,7 @@ export interface ToastProps extends Props, IntentProps {
 
     /**
      * Milliseconds to wait before automatically dismissing toast.
-     * Providing a value less than or equal to 0 will disable the timeout (this is discouraged).
+     * Providing a value less than or equal to 0 or Infinity will disable the timeout (this is discouraged).
```

Comment:
```suggestion
     * Setting this to Infinity or a value less than or equal to 0 will disable the timeout (this is discouraged).
```

Minor nit to make sure that this is clear that we're not referring to values less than infinity

Reply:
Great! Glad to help.